.github/workflows/oclif-release.yml

```diff
@@ -37,28 +37,24 @@ jobs:
             brew install makensis
           fi
           ${{ matrix.command_name }}
-      - name: Configure AWS credentials from Test account
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }}
-          aws-region: us-east-1
-      - name: Publish assests to s3 and calculate sha of oclif dev pack
+      - name: Calculate sha of oclif dev pack
         id: sha256
         if: ${{matrix.publish == 'homebrew'}}
         run: |
           make clean
           make install
-          npx oclif-dev publish
```

Comment:
Won't we still need to publish the darwin tarballs?

Reply:
Darwin tarballs will only be needed by `twilio update` cmd, which can fetch them successfully from the release assets. I don't think if there's a need to publish them to s3 bucket.